src/utils/p5PreloadHelper.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 ml5
```

Comment:
If the preloadHelper as-is isn't working as expected, then best to leave this file out.